build.env

```diff
@@ -41,7 +41,7 @@ VM_DRIVER=none
 CHANGE_MINIKUBE_NONE_USER=true
 
 # Rook options
-ROOK_VERSION=v1.4.9
+ROOK_VERSION=v1.6.2
 # Provide ceph image path
 ROOK_CEPH_CLUSTER_IMAGE=docker.io/ceph/ceph:v14.2.12
```

Comment:
This is the cause of `ceph: Mons are allowing insecure global_id reclaim` error. @humblec can you rebase on top of master ?